pipeline/container/frontend/src/components/play/DynamicFieldsForm.tsx

```diff
@@ -44,7 +37,7 @@ export function DynamicFieldsForm({
   } = formMethods;
 
   // Handlers
-  async function onSubmit(data: object) {
+  async function onSubmit(data: Record<string, any>) {
```

Comment:
Could be interesting to try and type the any out at some point, although may get hairy :p